src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -31,6 +30,13 @@
 __author__ = "Bill Riehl <wjriehl@lbl.gov>"
 __version__ = "0.0.1"
 
+JOB_FALSY_ERR = "Job ID is not provided or falsy"
+JOB_NOT_REG_ERR = "Job ID not registered"
```

Comment:
How about "Job ID is not registered" to keep in line with the rest of the strings.